src/test/java/org/zalando/nakadi/repository/kafka/NakadiKafkaConsumerTest.java

```diff
@@ -98,18 +98,18 @@ public void whenCreateConsumerThenKafkaConsumerConfiguredCorrectly() {
     public void whenReadEventsThenGetRightEvents() {
 
         // ARRANGE //
-        final String event1 = randomString();
-        final String event2 = randomString();
+        final byte[] event1 = randomString().getBytes();
+        final byte[] event2 = randomString().getBytes();
```

Comment:
@dehora I pushed it now. I had mistakenly pushed it to a different branch in my fork.